src/traces/scatter/plot.js

```diff
@@ -419,9 +419,6 @@ function plotOne(gd, idx, plotinfo, cdscatter, cdscatterAll, element, transition
         var enter = join.enter().append('path')
             .classed('point', true);
 
-        enter.call(Drawing.pointStyle, trace)
```

Comment:
@n-riesco thanks very much :tada: 

Your solution appears to work:
![peek 2017-05-23 11-43](https://cloud.githubusercontent.com/assets/6675409/26363199/5af464b6-3fae-11e7-80cc-8dd3d6b82c79.gif)



My only concern now is how this interacts with animations. I think the reason why `translatePoints` and `pointStyle` are called on enter selection **and** merge selection has to do with this [block](https://github.com/plotly/plotly.js/blob/cea9455ba353fe62ef374454cc954c4b7945bb0a/src/traces/scatter/plot.js#L425-L428) below where entering nodes have their opacity transitioned from 0 to 1. I suspect this commit will break this behavior (although the tests are passing) has the entering aren't positioned and styled properly before opacity transition cc @rreusser . It would be nice to write a test case here :smirk: 

I'm thinking instead, we could keep those `enter.call()`, make `Drawing.translatePoints` not (and maybe never) remove nodes and make the merge selection `.each` block below (which comes after the `.order()`) handle the cases where nodes need to be removed.

Thoughts?